### PR TITLE
Topic/license

### PR DESCRIPTION
**Pull request by GordanM**

### CODE REVIEW

**Comment by GordanM** [comment]
@caybro I am not sure if the `Author:` line is needed for every source file header. In my opinion you could state in the component README.md that you are the maintainer and the header `Author:` line could be removed, do you agree?

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Current backends:
...
 ## License
-LGPL v2.1
+Copyright (C) 2017 Pelagicore AB
 
-(c) 2017 Pelagicore AB / Luxoft
+The source code included here is licensed under the LGPL 2.1. Please
+see the "LICENSE" file for more information. Text documents are
+licensed under creative commons 4.0
+
+SPDX-License-Identifier: CC-BY-4.0
```
Comment (caybro) [inline README.md:19]: I don't think CC-BY-4.0 is correct here

**Review by caybro — CHANGES_REQUESTED** [review]
See the inline comment, otherwise :+1:

**Review by caybro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (caybro) [reply README.md:19]: nvm, I figured this applies to this file only :) all good

**Review by jeremiah — COMMENTED** [review]
Approved.

**Pull request merged**